lib/core/DocsLayout.js

```diff
@@ -34,15 +34,26 @@ class DocsLayout extends React.Component {
     const metadata = this.props.metadata;
     const content = this.props.children;
     const i18n = translation[this.props.metadata.language];
-    let DocComponent = Doc;
-    if (this.props.Doc) {
-      DocComponent = this.props.Doc;
-    }
     const title = i18n
       ? translation[this.props.metadata.language]['localized-strings'][
           this.props.metadata.localized_id
         ] || this.props.metadata.title
       : this.props.metadata.title;
+    const customLayouts = this.props.config.layouts;
+    let DocComponent = Doc;
+    if (customLayouts) {
+      if (customLayouts.default) {
+        DocComponent = customLayouts.default({
+          React,
+          MarkdownBlock,
```

Comment:
I think you forgot to import MarkdownBlock

You could do something like 
```js
MarkdownBlock: require('./MarkdownBlock.js')
```

or import it beforehand
```js
const MarkdownBlock = require('./MarkdownBlock.js');
```